R/verb-filter.R

```diff
@@ -15,11 +15,19 @@
 #' db %>% filter(is.na(x)) %>% show_query()
 # registered onLoad
 #' @importFrom dplyr filter
-filter.tbl_lazy <- function(.data, ..., .preserve = FALSE) {
+filter.tbl_lazy <- function(.data, ..., .by = NULL, .preserve = FALSE) {
   if (!identical(.preserve, FALSE)) {
     cli_abort("{.arg .preserve} is not supported on database backends")
   }
   check_filter(...)
+  by <- compute_by({{ .by }},
```

Comment:
Indenting style? You might want to try to http://github.com/lionel-/codegrip/ do toggle between tidyverse styles.